video-colorization/tcvc/neu_utils/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019 Sony Corporation. All Rights Reserved.
```

Comment:
I can't remember our discussion on neu, but I don't think you made change son everything, so you can probably load neu of nnabla-examples as is and keep the separate files here for the ones you did make changes only. (Maybe I said I would do it? I can't really remember)